File: Q00700-Q00799/Q00771_EASY/Q00771_Jewels_and_Stones.py
class Solution(object):
    def numJewelsInStones(self, jewels, stones):
        """
        :type jewels: str
        :type stones: str
        :rtype: int
        """
        counter = 0
        for i in stones:
            for j in jewels:
                if j == i:
                    counter += 1
        return counter

File: Q00700-Q00799/Q00771_EASY/Q00771_Jewels_and_Stones.java
class Solution {
    public int numJewelsInStones(String jewels, String stones) {
        char [] jewelArray = jewels.toCharArray();
        char [] stoneArray = stones.toCharArray();
        int counter = 0;
        for(int i = 0; i < stoneArray.length; i++){
            for(int j = 0; j < jewelArray.length; j++){
                if(stoneArray[i] == jewelArray[j]){
                    counter++;
                }
            }
        }
        return counter;
    }
}